### PR TITLE
ESPurna image

### PR DESCRIPTION
**Pull request by xoseperez**
Hi

I think it can be interesting to have the option to flash a basic ESPurna image without having to first download it from the release assets and copy it to the files folder.

This PR adds some documentation on how to do it and recommended steps after flashing it. The approach is the same as with the Tasmota image: a generic image you will then have to upgrade to a device-specific one.

If you think it's interesting, let me know how can I send you the binary to add to the repo. 

Thank you.

### CODE REVIEW

**Comment by kueblc** [comment]
Hi @xoseperez, I think this is a great idea. Big fan of your work. You can include the binary in the PR.

File: README.md [hunk 1]
```diff
@@ -77,22 +77,27 @@ BE SURE THE FIRMWARE FITS YOUR DEVICE!
...
-3. Start flashing process
+2. Start flashing process
 
         # curl http://10.42.42.42/flash3
 
-Alternatively you can request a certain file to be requested and flashed by the device:
+    Alternatively you can request a certain file to be requested and flashed by the device:
 
-	# curl http://10.42.42.42/flash3?url=http://10.42.42.1/files/thirdparty.bin
+        # curl http://10.42.42.42/flash3?url=http://10.42.42.1/files/thirdparty.bin
```
Comment (kueblc) [inline README.md]: What are these whitespace adjustments for?

File: README.md [hunk 2]
```diff
@@ -77,22 +77,27 @@ BE SURE THE FIRMWARE FITS YOUR DEVICE!
...
 
-4. Initial Configuration
+3. Initial Configuration
 
 	If you flashed the included Tasmota firmware file, it will broadcast a `sonoff-xxxx` access point (AP) when the device boots. Connect to this AP and open the browser to 192.168.4.1 to configure the device's Wi-Fi credentials. When entering the Wi-Fi password, click the checkbox to view the password you enter to ensure that it is correct and that your mobile device has not inadvertently capitalized the first letter if it is supposed to be lower case nor autocorrected what you entered. ~~Double~~ **Triple check the Wi-Fi credentials** before clicking **Save** to apply the settings.  
 
+    If you flashed the included ESPurna fimaware file, the procedure will be very similar. The device will broadcast a `ESPURNA-XXXXXX` access point. You will have t connect to it using the default password: `fibonacci`. Once connected open the browser to 192.168.4.1 and follow the initial configuration instructions. Then go to the WIFI tab and configure your home WiFi connection (remember to save) or go to the ADMIN tab to upgrade the firmware to the device-specific image.
```
Comment (kueblc) [inline README.md]: `You will have t connect` to `You will have to connect`

**Comment by kueblc** [comment]
Let's rename `thirdparty.bin` back to `sonoff.bin` and make `thirdparty.bin` a symlink again. I think this was inadvertently changed in #277, but more important now if we are including multiple options.

File: README.md (hunk 1 above)
Comment (xoseperez) [reply README.md]: Indentation looked wrong in the preview.

**Comment by digiblur** [comment]
Would be cool to have a little menu driven type thing for people in the script with confirmations.  Have a few bins and let them pick out of things.  There are the purists that still want a Tasmota built bin and understand the risks of the one time WiFi password shot so I would say to keep that one.

**Comment by kueblc** [comment]
> Would be cool to have a little menu driven type thing for people in the script with confirmations. Have a few bins and let them pick out of things.

This is in the works :smile: 

> There are the purists that still want a Tasmota built bin and understand the risks of the one time WiFi password shot so I would say to keep that one.

Good points

**Comment by kueblc** [comment]
Thanks @xoseperez LGTM

**Pull request merged**